### PR TITLE
product card solution

### PR DESCRIPTION
**Pull request by APetrinko**
- [DEMO LINK](https://APetrinko.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://APetrinko.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

**Comment by APetrinko** [comment]
You can review.
The mentor knows about issue with test

**Review by AntonMartynes — APPROVED** [review]
Good job!

File: styles/card.css [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
```
Comment (ivan-kadykalo) [inline styles/card.css:75]: [Read this link](https://www.w3schools.com/html/html_blocks.asp)
```suggestion
```

File: styles/star.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(/src/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 .stars__star:first-child,
+.stars--2 .stars__star:nth-child(-n+2),
```
Comment (ivan-kadykalo) [inline styles/star.css:18]: Duplicated file

File: styles/card.css [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+  display: block;
+}
+
+.card__name {
+  margin-bottom: 4px;
+  font-style: normal;
```
Comment (ivan-kadykalo) [inline styles/card.css:18]: Duplicated parameter. Use it in your reset file for whole page

**Review by ivan-kadykalo — CHANGES_REQUESTED** [review]
This problem could be because of the wrong font family or weight.

Probably now are working only fallback font 'sans-serif', try to delete sans-serif from fallback and you will see that Roboto doesn't work.

image

**Pull request closed without merging**